src/frontend/packages/kubernetes/src/helm/list-types/monocular-chart-card/monocular-chart-card.component.ts

```diff
@@ -12,7 +16,16 @@ export class MonocularChartCardComponent extends CardCell<MonocularChart> {
 
   @Input() row: MonocularChart;
 
+  public artifactHubAndOthers$: Observable<boolean>;
+
   constructor() {
     super();
+    this.artifactHubAndOthers$ = stratosEntityCatalog.endpoint.store.getAll.getPaginationService().entities$.pipe(
+      filter(entities => !!entities),
+      first(),
+      map(endpoints => !!endpoints.find(ep => ep.sub_type === HELM_HUB_ENDPOINT_TYPE) && endpoints.length > 1),
```

Comment:
If we mean to only show when more than one Helm endpoint is available, we need to filter the endpoints to those that are only Helm before checking the length